mithril-client/src/client.rs

```diff
@@ -108,14 +108,20 @@ impl Client {
         self.mithril_stake_distribution_client.clone()
     }
 
+    #[deprecated]
```

Comment:
Adding a note would help users to switch
```suggestion
    #[deprecated(since = "0.11.9", note = "supersede by `cardano_database`")]
```

Reply:
Done
I do the same in `client-wasm` for `get_snapshot` and `list_snapshots